### PR TITLE
feat : added routes to pages

### PR DESCRIPTION
**Pull request by Digambar1411**
added routes to all pages

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,43 +1,22 @@
...
-            Contribute
-          </a>
-        </div>
-      </header>
+      <Routes>
+        <Route path="/" element={<Home />}/>
+        <Route path="/login" element={<Login />}/>
+        <Route path="/signup" element={<Signup />}/>
+        <Route path="/LV" element={<LikedVideos />}/>
+        <Route path="/WL" element={<WatchLater />}/>
```
Comment (ranitasahaa1230) [inline src/App.js]: Suggestion: Instead of writing LV, WL, write `liked-videos`, `watch-later`. It's a good practice

**Review by ranitasahaa1230 — APPROVED** [review]
Code looks clean
Good to go

**Review by surbhi84 — APPROVED** [review]
- Code looks neat.
- plus one on Ranita's suggestion.
- Go ahead with the merge.

Just a suggestion: You can add a whole feature in a single pr, that way you will have to get less number of pr's reviewed as well.

**Comment by Digambar1411** [comment]
> * Code looks neat.
> * plus one on Ranita's suggestion.
> * Go ahead with the merge.
> 
> Just a suggestion: You can add a whole feature in a single pr, that way you will have to get less number of pr's reviewed as well.

yeah doing the same. but i m working on another project until then did this setup for this project

**Pull request merged**